LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationNetworkServerStartup.cs

```diff
@@ -125,6 +125,10 @@ public void ConfigureServices(IServiceCollection services)
 
             if (NetworkServerConfiguration.ClientCertificateMode is not ClientCertificateMode.NoCertificate)
                 _ = services.AddSingleton<IClientCertificateValidatorService, ClientCertificateValidatorService>();
+            if (NetworkServerConfiguration.RunningAsIoTEdgeModule)
+            {
+                _ = services.AddSingleton<ModuleConnectionHost>();
```

Comment:
On L82 we unconditionally register the `ModuleConnectionHost` (I think this came from a merge conflict). Let's make sure to remove it there.